.github/workflows/pull-request.yml

```
@@ -449,3 +449,91 @@
+          if [ ! -f docs/docs/developer-guide/intro.md ]; then
+            echo "# Developer Guide\n\nThis section contains the developer guide for Talawa Admin." > docs/docs/developer-guide/intro.md
+          fi
+          if [ ! -f docs/docs/developer-guide/reference/README.md ]; then
+            echo "# API Reference\n\nThis section contains the auto-generated API documentation for Talawa Admin." > docs/docs/developer-guide/reference/README.md
+          fi
+
+      # Step 6: Commit Documentation Changes
+      - name: Commit Documentation Changes
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
```

actionlint (1.7.4)

"github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)